frontend/client/src/components/NavBar.js

```diff
@@ -21,15 +21,17 @@ const useStyles = makeStyles((theme) => ({
   title: {
     flexGrow: 1,
   },
+  link: {
+    marginTop: 20,
```

Comment:
I believe this should only be 14px.  The Figma shows a total vertical space of 20px between the each text option in the menu.  There is already 6px of padding, so I think 14px will achieve the total of 20px.